### ISSUE TITLE
Pack and Go-Go Error

### ISSUE THREAD

**Issue by mjjvc** [macmarrum/freeplane-PackAndGoGo#3]
Hi
It's not working for me. Just installed the Add-On.
Freeplane v. 1.11.7
Java v. 15.0.10
Pack and Go-Go 2023.10.05

I will attach the Map, Images and Screenshot of the Error.

I tried it 3 times.
1.
Resulted with Error and a Zip file. After opening the zip file all the images were included except the last image on the last node. 
2.
The last 2 nodes were open when I packed it. 
Resulted in no zip file appearing.
3.
New mind map without any images.
Map was saved as a zip file.

[Pack 'n Go Go Map - Files & Sreenshot of Error.zip](https://github.com/macmarrum/freeplane-PackAndGoGo/files/13034523/Pack.n.Go.Go.Map.-.Files.Sreenshot.of.Error.zip)

Thanks.

**Comment by macmarrum** [comment]
@mjjvc 
The attached zip contains a mind map with absolute links. This makes it very hard to use it for testing. I am having trouble reproducing the error.

Could you please try the latest release and report back? It contains some fixes, mostly related to
- #2

**Comment by mjjvc** [comment]
Hi
Freeplane ver. 1.11.7
Java ver 15.010
Pack and Go Go ver 2023.10.19

I think they are the latest versions. 

I have taken a screenshot of the map with  Hyperlink relative links option selected. I did not save the preferences to a file. The preference is set to relative when I open the map and check the Hyperlink types.

Is there a different Relative/Absolute setting for Images or does the Hyperlinks Relative/Absolute setting work for images also?

I also copied the map and images into a different location. I right-clicked on one of the images in the new map and checked the file location and it is loaded from the new location.


[Pack and Go Go Error Files Zip.zip](https://github.com/macmarrum/freeplane-PackAndGoGo/files/13131101/Pack.and.Go.Go.Error.Files.Zip.zip)

There are several screenshots of the error, versions and Relative Option selected.

Hope this helps,

Thanks.
